js/aside-script.js

```diff
@@ -211,7 +220,18 @@ function UpdateLocale()
 	);
 }
 
-function AddCollection(collection)
+
+function ReloadCollections(collections,thumbnails){
+				document.querySelector(".tabsAside section h2").removeAttribute("hidden");
+				document.querySelectorAll(".tabsAside section > div").forEach(i => i.remove());
+
+				if (document.querySelector(".tabsAside.pane section > div") == null)
+
+					for (const collection of Object.values(collections))
```

Comment:
Why do we need to `Object.values()` wrapper?

Reply:
Because we can't iterate directly over Objects like that in JS, Objects are not iterable. 
So we have to specify that we iterate over the array of values.

Here's the result in the DevTools console : 
![image](https://user-images.githubusercontent.com/15179425/100544168-2a1b3580-3254-11eb-9790-00a1df7ee747.png)

Note that `for (x in collection)` would be looping over keys/properties in this case

Reply:
Wait, but our `collections` variable contains an array. And I've checked, it does work without this wrapper

![image](https://user-images.githubusercontent.com/28831743/100545033-c7816400-326a-11eb-90e1-585874463616.png)

Reply:
You're right - I definitely got confused between `collection `and `collections`. So that should apply to your other comment regarding length. I'll fix that, thanks for the heads-up !!

Reply:
Ah, wait, no. Are you on the right branch ? I see your collection item has an "icons" key, which I actually deleted ? Give me a few minutes and I'll get back to you after verifying all that array/object stuff

Reply:
Nope, my approach indeed uses a collection object, and not an array. There is also no way for a collection to have an `icons` key in the current code; so I think there is a branch mixup on your end. Maybe your screenshot was done from the store version of the extension ?

I believe my first comment still stands, here is it on this PR's branch today, and since it is an object it needs the wrapper : 
![image](https://user-images.githubusercontent.com/15179425/100545514-7e2a1800-325c-11eb-98bd-5c517bd674ad.png)

Reply:
Yeah, I've missed that part. Makes sense